ckanext/ontario_theme/plugin.py

```diff
@@ -1122,7 +1160,38 @@ def organization_facets(self, facets_dict, organization_type, package_type):
     def before_search(self, search_params):
         u'''Extensions will receive a dictionary with the query parameters,
         and should return a modified (or not) version of it.
+        Sets default access level to open
         '''
+        fl = search_params.setdefault("facet.field", [])
+        fq = search_params.get("fq", "")
+        default_open = '{!bool tag=orFqaccess_level should=\'access_level:"open"\'}'
+
+        ors = set(_get_default_ors())
+        # To show all access levels on dataset counts on
+        # the homepage and org/group pages
+        if ('fq' not in search_params) or (fq and not fl):
+            fq_list = search_params.setdefault('fq_list', [])
+        # Show default open datasets or other access level on org/group search
+        # pages
+        elif fq:
+            fq_list = search_params.setdefault('fq_list', [default_open])
+            access_level_count = True if fq.count("access_level") > 1\
+                or (fq.count("access_level") == 1 and 'access_level:"open"' not in fq) else None
```

Comment:
`fq.count("access_level") > 1 or (fq.count("access_level") == 1` can be simplified to `fq.count("access_level") >= 1`

Reply:
Thanks @aleeexgreeen ! This solves the problem. I didn't notice the brackets around the second `or` grouping! I'll revert